shm/cargo.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include "subjectTime.hpp"
+#include "time.hpp"
```

Comment:
I think there should be a new line between standard library and headers include.

Reply:
it's a matter of the group internal convention and you should not comment the coding style in group projects.